geometry_msgs/Quaternion relative_rotation
---
bool success